internal/monitors/expvar/metadata.yaml

```diff
@@ -156,6 +157,7 @@
   - name: memstats.heap_sys
     type: gauge
     description: Bytes of heap memory obtained from the OS
+    included: false
```

Comment:
What's up with this?

Reply:
It was just missing for some reason. Defaults to false but test code assumes they're set explicitly.